### PR TITLE
Use provider.getStackName method to get stackname

### LINKED ISSUE #15: Custom stackName in next serverless version that will brake this plugin.
In serverless version above 1.27.3 one can set a custom stackName. 
 ```
provider:
    stackName: my-custom-name
```
In main serverless they are using this getStackName from this code to fetch the stackname:
https://github.com/serverless/serverless/blob/cbc5e3c4e199a08650b02def8d1f9e91a13e7844/lib/plugins/aws/lib/naming.js

Used as in serverless, if you get the this reference correct you are done:
```
const stackname = this.provider.naming.getStackName();
```

### PR DESCRIPTION
**Pull request by JakeChampion**
Fixes #15

### CODE REVIEW

File: src/plugin.ts [hunk 1]
```diff
@@ -27,10 +27,7 @@ export default class StackOutputPlugin {
   }
 
   get stackName () {
-    return util.format('%s-%s',
-      this.serverless.service.getServiceName(),
-      this.serverless.getProvider('aws').getStage()
-    )
+    return this.serverless.getProvider('aws').naming.getStackName()
```
Comment (aregier) [inline src/plugin.ts:30]: I've seen this same behavior, too. Out of curiosity, why not just read directly from the property (e.g. `this.serverless.getProvider('aws').stackName`) and declare that in the typings file?

This this value be populated if the stackName property is *not* set in serverless.yml? It's possible that we need to fall back to the defaulted value here?

Comment (aregier) [reply src/plugin.ts:30]: nm; It looks like it is fine even if stackName is not specified in serverless.yml

**Comment by egut** [comment]
I apparently missed somethings but it seams to be fixed. Any plan to merge this?

**Comment by viyh** [comment]
+1, just hit this bug.

**Pull request closed without merging**